build/android/native/build.gradle

```diff
@@ -0,0 +1,60 @@
+apply plugin: 'com.android.library'
+import org.ajoberstar.grgit.Grgit
+
+android {
+	compileSdkVersion 29
+	buildToolsVersion '29.0.2'
+	defaultConfig {
+		minSdkVersion 16
+		targetSdkVersion 29
+		externalNativeBuild {
+			ndkBuild {
+				arguments '-j',
+						'APP_STL=c++_static',
+						'COMPILER_VERSION=clang',
+						'APP_SHORT_COMMANDS=true',
+						"versionMajor=${versionMajor}",
+						"versionMinor=${versionMinor}",
+						"versionPatch=${versionPatch}"
+			}
+		}
+	}
+
+	externalNativeBuild {
+		ndkBuild {
+			path file('jni/Android.mk')
+		}
+	}
+
+	// supported architectures
+	splits {
+		abi {
+			enable true
+			reset()
+			include 'armeabi-v7a', 'arm64-v8a'//, 'x86'
+		}
+	}
+
+	buildTypes {
+		release {
+			externalNativeBuild {
+				ndkBuild {
+					arguments 'NDEBUG=1'
+				}
+			}
+		}
+	}
+}
+
+task cloneGitRepo() {
```

Comment:
When I understand it right, what's about `git submodules`, to get rid of this ?

Reply:
This is a very large repository. Those who do not plan to build an Android version are unlikely to want to clone it. This is the best solution.

Reply:
Yes and no.
Keep in mind a normal 
`git clone https://github.com/minetest/minetest.git`
does nothing with submodules, you have to explicit perform a
`git submodule update --init`
Doing so, it's an optional step